doc/reference/backends.rst

```diff
@@ -2,18 +2,32 @@
 Backends and Configs
 ********************
 
-.. note:: Both NetworkX backend and config systems are experimental. They
-   let you execute an alternative backend implementation instead of NetworkX's
-   pure Python dictionaries implementation. Things will almost certainly
-   change and break in future releases!
+.. note:: Both NetworkX backend and config systems are experimental.
```

Comment:
I like the new separate sections in the code for backend users and backend developers, but people reading this doc may not see that and think backends should not be used for production code.  Since this doc seems to be aimed at end users, my preference would be to remove this note completely.